src/onenet_mqtt.c

```diff
@@ -84,54 +104,77 @@ static rt_err_t onenet_mqtt_entry(void)
     mq_client.defaultMessageHandler = mqtt_callback;
 
     paho_mqtt_start(&mq_client);
-    
+
     return RT_EOK;
 }
 
-static void onenet_get_info(void)
+static rt_err_t onenet_get_info(void)
 {
-    strncpy(onenet_info.device_id, ONENET_INFO_DEVID, strlen(ONENET_INFO_DEVID));
-    strncpy(onenet_info.api_key, ONENET_INFO_APIKEY, strlen(ONENET_INFO_APIKEY));
+    char dev_id[ONENET_INFO_DEVICE_LEN] = { 0 };
+    char api_key[ONENET_INFO_DEVICE_LEN] = { 0 };
+    char auth_info[ONENET_INFO_DEVICE_LEN] = { 0 };
+
+    if (onenet_port_get_device_info(dev_id, api_key, auth_info))
+    {
+        log_e("onenet get device id fail,dev_id is %s,api_key is %s,auth_info is %s\n");
```

Comment:
log_e("onenet get device id fail,dev_id is %s,api_key is %s,auth_info is %s\n"，dev_id，api_key， auth_info );